src/index.js

```diff
@@ -1,19 +1,58 @@
-import _ from 'lodash';
-import printMe from './print.js';
+// import _ from 'lodash';
```

Comment:
- [OPTIONAL] Kindly remove all commented line code, It is always a good practice to keep your code clean from such comments 💯 👍🏻